content/en/real_user_monitoring/mobile_and_tv_monitoring/android/advanced_configuration.md

```diff
@@ -397,6 +345,12 @@ You can use the following methods in `RumConfiguration.Builder` when creating th
 `setErrorEventMapper`
 : Sets the EventMapper for the RUM ErrorEvent. you can use this interface implementation to modify the ErrorEvent attributes before serialization.
 
+`setInitialResourceIdentifier`
+: Sets a custom identifier for initial network resources used for [Time-to-Network-Settled][11] (TNS) view timing calculation.
+
+`setLastInteractionIdentifier`
+: Sets a custom identifier for the last interaction in the previous view used for [Interaction-to-Next-View[14]] (INV) timing calculation.
```

Comment:
There is a problem with the `[14]` link - syntax is wrong and the link is not defined at EOF.